tests/tests.cpp

```diff
@@ -1863,7 +1864,7 @@ TEST_F(EngineBasicTest, TestExpireAPI) {
 
   std::string got_val;
   int64_t ttl_time;
-  WriteOptions write_options1{0 /* invalid argument */, false};
+  WriteOptions write_options1{1, false};
```

Comment:
1 for what?

Reply:
ttl_time is 1s. 0 is invalid, similar to redis.